src/main/menu_app/src/components/Navbar.js

```diff
@@ -1,25 +1,24 @@
 import React from "react";
-import { NavLink } from "react-router-dom";
+import { Link } from "react-router-dom";
 
 import './../App.css';
```

Comment:
Looks like this should be removed

Reply:
For me it doesn't make sense... I'm pretty sure I'll need it in the nearest future. But I agree that on this step it looks like an extra line.

Reply:
Oh, I misunderstood what happened here. I thought that the `App.css` was removed completely.
If it is just empty now, then this line can stay, I guess.
IMO, I'd drop an empty file and this line all together, yes, but I guess it is not that significant for now